ipa-core/src/helpers/transport/query/mod.rs

```diff
@@ -220,8 +222,95 @@ impl AsRef<str> for QueryType {
         }
     }
 }
+#[cfg(test)]
+impl PartialEq for IpaQueryConfig {
+    fn eq(&self, other: &Self) -> bool {
+        self.per_user_credit_cap == other.per_user_credit_cap
+            && self.max_breakdown_key == other.max_breakdown_key
+            && self.attribution_window_seconds == other.attribution_window_seconds
+            && self.num_multi_bits == other.num_multi_bits
+            && self.dp_params == other.dp_params
+            && self.plaintext_match_keys == other.plaintext_match_keys
+    }
+}
+
+// TODO switch to using an enum
+#[derive(Debug, Copy, Clone, Serialize, Deserialize)]
+pub enum DPParams {
```

Comment:
Naming nit: Don't like the DP acronym too much. Also "params" doesn't add a lot of substance to the name. I don't know the space well enough to provide a better name.

Why not just have functions pass an `Option<DifferentialPrivacy>` instead of having an explicit no enum? If you use an Option you will get all those helper functions for free.

Reply:
Maybe we should follow MS convention and use `Dp` instead. `DpParams` looks better

Reply:
Acronyms are words in [the rust style](https://rust-lang.github.io/api-guidelines/naming.html).  So `DpParams` it is.

Reply:
Sounds good. changing to `DpParams` also `WithDp`, `NoDp`